lib/pbench/server/api/__init__.py

```diff
@@ -84,4 +85,10 @@ def create_app(server_config):
 
     register_endpoints(api, app, server_config)
 
+    Database.init_db(server_config=server_config, logger=app.logger)
+
+    @app.teardown_appcontext
+    def shutdown_session(exception=None):
+        Database.db_session.remove()
```

Comment:
This would happen only once in the rundown of the API within each gunicorn worker thread, so I don't think it's a big issue. Again, I just stole this from Nikhil and, I admit, I didn't spent any time researching the semantics. If we're going to touch stuff like this it'd be best to reflect it back to #1937 and then merge it here.